libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -461,7 +463,24 @@ bool AP_RCProtocol_CRSF::decode_crsf_packet()
             break;
     }
 #if HAL_CRSF_TELEM_ENABLED
-    if (AP_CRSF_Telem::process_frame(FrameType(_frame.type), (uint8_t*)&_frame.payload)) {
+    if (AP_CRSF_Telem::process_frame(FrameType(_frame.type), (uint8_t*)&_frame.payload, _frame.length - 2U)) {
+#ifdef CRSF_DEBUG_TELEM
+        switch (_frame.type) {
+            case CRSF_FRAMETYPE_RC_CHANNELS_PACKED:
+            case CRSF_FRAMETYPE_LINK_STATISTICS:
+            case CRSF_FRAMETYPE_SUBSET_RC_CHANNELS_PACKED:
+            case CRSF_FRAMETYPE_LINK_STATISTICS_RX:
+            case CRSF_FRAMETYPE_LINK_STATISTICS_TX:
+                break;
+            default:
+                hal.console->printf("CRSF: received %s:", get_frame_type(_frame.type));
```

Comment:
```suggestion
                GCS_SEND_TEXT(MAV_SEVERITY_INFO, "CRSF: unknown frame %s:", get_frame_type(_frame.type));
```

Reply:
This is deliberate and is compiled in debug, the frame is not unknown - we just want to see it.